### PR TITLE
Change preset loading confirmation message to say that settings will be overwritten

### PR DESCRIPTION
**Pull request by DNin01**
Resolves no open issues.

### Changes

Are you sure you want to load this preset? → Load this preset and overwrite current settings?

### Reason for changes

Someone may not realize that loading a preset is going to permanently overwrite their custom settings.

### Tests

Shouldn't need a test.

### CODE REVIEW

**Comment by apple502j** [comment]
Hmm, does it actually?

**Comment by WorldLanguages** [comment]
> Hmm, does it actually?

Does it actually what?

**Comment by DNin01** [comment]
> Hmm, does it actually?

Actually overwrite the settings?

**Review by RedGuy12 — APPROVED** [review]

**Review by WorldLanguages — APPROVED** [review]

File: _locales/en/messages.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "message": "For website"
   },
   "confirmPreset": {
-    "message": "Are you sure you want to load this preset?"
+    "message": "Load this preset and overwrite current settings?"
```
Comment (WorldLanguages) [inline _locales/en/messages.json:69]: We could possibly change it to `current addon settings` to clarify that settings from other addon settings can't be affected by setting a preset. What do you think?

Comment (DNin01) [reply _locales/en/messages.json]: Done.

Comment (WorldLanguages) [reply _locales/en/messages.json]: @RedGuy12 Please confirm if you still approve this

**Review by apple502j — APPROVED** [review]

**Pull request merged**